lms/static/grader.js

```diff
@@ -52,12 +52,18 @@ function sendComment(kind, fileId, line, commentData) {
 function visuallyRemoveComment(commentId) {
```

Comment:
Maybe we should also remove the trash icon from comments that aren't belong to the user?

Reply:
Only staff are going to see all the trashes

Reply:
Users will see their trash only

Reply:
Look at `isSolverComment` function in `comments.js`